tools/clang/tools/dxil2spv/lib/dxil2spv.cpp

```diff
@@ -336,11 +350,21 @@ void Translator::createLoadInputInstruction(llvm::CallInst &instruction) {
                                   hlsl::DXIL::OperandIndex::kLoadInputIDOpIdx))
           ->getLimitedValue();
   spirv::SpirvVariable *inputVar = inputSignatureElementMap[inputID];
+  if (!inputVar) {
+    emitError(
+        "No matching SPIR-V input variable found for load instruction: %0",
+        instruction);
```

Comment:
Does LLVM support formatting `llvm::CallInst &instruction`?

Reply:
Yes, this `emitError` wrapper calls the `llvm::Value::print` function to pretty-print it.